druid/examples/transparency.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2019 The Druid Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! An example of a transparent window background.
+//! Useful for dropdowns, tooltips and other overlay windows.
+
+use druid::{kurbo::{Circle}, widget::{Button, Flex}};
```

Comment:
I dont know the druid guidelines around this, but I think its preferred to not have nested imports like this?

Reply:
Yeah, I removed that, VSCode does these automatically, thanks!